README.md

```diff
@@ -229,6 +229,37 @@ The server provides following default HTTP(s) endpoints
 - http://localhost:8080/__info/php-opcache
 - http://localhost:8080/__info/php-xdebug (only if xdebug is enabled)
 
+## Optional services 
+
+In order to provide you with a minimal, fast local environment; we've opted to not initialise the joomlatools-server with the following options: 
+
+* Dnsmasq 
+* Traefix 
+* Minica 
+* Phpmyadmin
+* Mailhog
+* Dozzle 
+
+In order to run any of these services, at start up, all you need to do is provide the profile option during the compose up: 
+
+`docker compose --profile mailhog up -d`
+
+For instance, this will start the Mailhog service, which will let you intercept emails sent from your application in a handy to use graphical interface available at: http://localhost:8380/
+
+The [docker-compose.yml](https://github.com/joomlatools/joomlatools-server/blob/master/docker-compose.yml#L192) file is documented as to how to start and control each of these additional profiles. 
+
+If you wanted to run all of these services at once: 
+
+`docker compose --profile all up -d `
```

Comment:
hmm docker compose commands, when we have:
https://github.com/joomlatools/joomlatools-server/blob/master/bin/jtctl

Establish how to provide profile arguments into this command and remove references to `docker compose`

Reply:
sh bin/jtctl start

Actually loads up all available profiles:
https://github.com/joomlatools/joomlatools-server/blob/master/bin/jtctl#L183

So this issue/ Pull request is not needed